src/download.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { verifyStarterExists } from './download';
+
+describe('download', () => {
+  describe('verifyStarterExists', () => {
+    it('returns false if starter does not exist', async () => {
+      expect(await verifyStarterExists({ repo: 'foo/bar' } as any)).toBe(false);
```

Comment:
Do these tests mock out `request` at all? I don't believe we have anything in this project that prevents network requests in unit tests at the moment (sadly), but could be mistaken

Reply:
They don't. I could mock out the `https` module and prevent going through the network but I thought it might be useful to include this extra loop to verify that the starter template is actually accessible. I don't believe it should cause any issues.

Reply:
My only concern here is lack of internet connection - I've been doing quite a bit of offline development this summer due to travel, which is why my mind goes there. Happy to try this out and see how founded those fears are in reality though

Reply:
Ok, let me know. I am fine mocking it out.